src/classes/CRLP_RollupQueryBuilder.cls

```diff
@@ -253,11 +264,11 @@ public without sharing class CRLP_RollupQueryBuilder {
      * DetailRecordId's are required to generate this query.
      * @return SOQL String
      */
-    public String getRelatedRecordsQueryAsSOQL() {
+   /* public String getRelatedRecordsQueryAsSOQL() {
```

Comment:
Should these commented out methods be deleted or is there a reason to keep them here?

Reply:
I was leaving them here because it's a helpful reminder that the query builder service needs to be expanded to build the related records query. I only completed about half of the query building re-write in this WI/PR. There's other query logic in the `CRLP_Query_SEL` and `CRLP_Batch_Base` classes that I'd eventually like to consolidate.